### DISCUSSION TITLE
OVERTIME

### DISCUSSION THREAD

**Discussion by gameinside** [SmileYzn/PugMod-AMXX#31] [Ideas]
I found one bug with overtime, for example, they need to work 3 to win, switch teams and win one round to completely win the match

Now the plugin only works to win 3 rounds 

pug_play_overtime_rounds "4"  ; you only play 2 rounds 

Maybe I didn't understand something
I'm sorry I'm just a girl

**Comment by SmileYzn** [comment]
I never in really count that. need to check it again the overtime method.
Also need someone that know how that works, since the difference to win is 2 rounds.

Do not worry, any help an idea is welcome :pray:

**Reply by gameinside** [reply]
The principle is, for example, there are two  TEAM  A-B "A is T!   B is CT" Team A Won with a score of 18:15, they change sides if team A won one round for CT" or rather, all 4 rounds she won the match

**Reply by SmileYzn** [reply]
And if in overtime the match tie in 3x3 the overtime will start again right?

**Reply by gameinside** [reply]
> And if in overtime the match tie in 3x3 the overtime will start again right?

yes

**Comment by SmileYzn** [comment]
Fixed.

Thanks for testing it

**Reply by gameinside** [reply]
cool it's works. tnx =* I didn't doubt you

**Reply by gameinside** [reply]
I wanted to ask, can you make PUBSTATS like statsx + shell???? demo https://c-s.net.ua/forum/topic73300.html
But what would PUGranks remain unchanged only + SKILL was added?

**Comment by SmileYzn** [comment]
This is a offline stats?
I'm creating a stats that support local database and remote database.

**Reply by gameinside** [reply]
> This is a offline stats? I'm creating a stats that support local database and remote database.

there is another addition to this module https://forums.alliedmods.net/showthread.php?t=279483

https://drive.google.com/file/d/1flw02mBFNpN7zTONPwDjlNbC2oTIev2B/view?usp=sharing look look at this file
